projects/ngrx.io/content/guide/data/index.md

```diff
@@ -0,0 +1,143 @@
+# NgRx Data
+
+NgRx Data is an extension that offers a gentle introduction to NgRx by simplifying management of **entity data** while reducing the amount of boilerplate.  
+
+Entity data is only one kind of application data.  NgRx Data should not be used for non-entity or highly idiosyncratic data.
+
+<div class="alert is-important">
+
+An **entity** is an object with long-lived data values that you read from and write to a database.  An entity refers to some "thing" in the application domain.  Examples include a _Customer_, _Order_, _LineItem_, _Product_, and _User_.
+
+</div>
+
+## Introduction
+
+Many applications have substantial _domain models_ with 10s or 100s of entity types.
+
+In NgRx, to create, retrieve, update, and delete (CRUD) data for every entity type is an overwhelming task. You're writing _actions_, _action-creators_, _reducers_, _effects_, _dispatchers_, and _selectors_ as well as the HTTP GET, PUT, POST, and DELETE methods _for each entity type_.
+
+In even a small model, this is a ton of repetitive code to create, maintain, and test.
+
+NgRx Data allows consumers to use NgRx while radically reducing the boilerplate necessary to manage entities with NgRx.
+
+## Key Concepts
+
+- NgRx Data can be used with NgRx to provide simplified management of entity data.
+- NgRx Data automates the creation of actions, reducers, effects, dispatchers, and selectors for each entity type.
+- NgRx Data provides default HTTP GET, PUT, POST, and DELETE methods for each entity type.
+- NgRx Data should not be used for non-entity or highly idiosyncratic data.  They are better managed with NgRx.
+
+## Defining the entities
+
+We create an `EntityMetadataMap` to tell NgRx Data about our entities.  Add a property to the set for each entity name.
```

Comment:
Change the references from `We` and `our` to `You` and `your`

Reply:
Updated in 5ef16b0.